src/scripts/site.js

```diff
@@ -32,23 +32,23 @@ function getParameterByName(name, url) {
 function pageClassOn(eventName, className, fn) {
     "use strict";
 
-    document.addEventListener(eventName, function (e__q) {
-        var target = e__q.target;
+    document.addEventListener(eventName, function (event) {
```

Comment:
Just an fyi, `event` is also a global variable. Not that it matters here...

Reply:
it's a global variable only for a few selected browsers. For example, firefox don't define event as a global variable, so it's safer to use the event passed as argument.

Reply:
I was more hinting at renaming the variable actually since I dislike shadowing

Reply:
Oh i see, but is the same thing here. Window.event is equal to that variable. Still do you want me to change the name?